corehq/apps/users/tests/retire.py

```diff
@@ -248,7 +248,7 @@ def test_multiple_case_blocks_some_deleted(self, rebuild_case):
                 owner_id=owner_id,
                 user_id=self.commcare_user._id,
             )
-            submit_case_blocks(caseblock.as_string(), self.domain, user_id=self.other_user._id)
+            submit_case_blocks(caseblock.as_string().decode('utf-8'), self.domain, user_id=self.other_user._id)
```

Comment:
would it be hard to make `caseblock.as_string()` return text rather than bytes?

Reply:
I'm going to keep a running list of parts of the code that work but could be cleaned up via unicode sandwiching:

https://trello.com/c/1VR9U4vh/73-unicode-sandwich

I'll plan on making these updates after the travis build is passing in py3.